lib/src/main/java/org/exoplatform/social/addons/updater/ActivityMigrationService.java

```diff
@@ -215,23 +208,14 @@ private long migrateSpaceActivities() throws Exception {
           Node node = null;
 
           while(it.hasNext()) {
-            if (forceStop) {
+            if (forkStop) {
```

Comment:
No idea, just tried to reuse one on parent class instead of introduce to new one. I will see and rename this property.